bundles/AdminBundle/Resources/public/js/pimcore/object/tags/booleanSelect.js

```diff
@@ -34,9 +34,11 @@ pimcore.object.tags.booleanSelect = Class.create(pimcore.object.tags.abstract, {
                 }
             }
 
-            for(var i=0; i<field.layout.options.length; i++) {
-                if(field.layout.options[i]["value"] == value) {
-                    return field.layout.options[i]["key"];
+            if(field.layout.options !== undefined) {
```

Comment:
There should be a space between `if` and `(` (and preferably also between `for` and `(` below).